_posts/2000-01-03-redundancy-payment.markdown

```diff
@@ -3,6 +3,7 @@
 layout: exemplar
 
 
+exemplar-service-url: "https://notice.redundancy-payments.org.uk/claims/start"
```

Comment:
Confirmed - there is no GOV.UK start page for this service. What's our view? Take it out or keep it in? I don't have an opinion here.